railties/lib/rails/generators/rails/app/templates/Dockerfile.tt

```diff
@@ -70,7 +70,7 @@ FROM base
 <% unless dockerfile_deploy_packages.empty? -%>
 # Install packages needed for deployment
 RUN apt-get update -qq && \
-    apt-get install --no-install-recommends -y <%= dockerfile_deploy_packages.join(" ") %> && \
+    apt-get install --no-install-recommends -y curl <%= dockerfile_deploy_packages.join(" ") %> && \
```

Comment:
Perhaps curl should be added to the [dockerfile_deploy_packages](https://github.com/rails/rails/blob/1545a8d74b08ecd407afbb9467c202f279da377b/railties/lib/rails/generators/app_base.rb#L558) list?

Reply:
Updated, thanks.